src/ui/public/field_editor/field_editor.js

```diff
@@ -24,12 +24,13 @@ uiModules
       getField: '&field'
     },
     controllerAs: 'editor',
-    controller: function ($scope, Notifier, kbnUrl) {
+    controller: function ($scope, Notifier, kbnUrl, es) {
       let self = this;
       let notify = new Notifier({ location: 'Field Editor' });
 
       self.scriptingInfo = scriptingInfo;
       self.scriptingWarning = scriptingWarning;
+      self.scriptingLangs = getScriptingLangs();
```

Comment:
`getScriptingLangs` isn't actually returning anything, so there's no point in assigning it to `self.scriptingLangs`.